### PR TITLE
MKS-43 Create endpoint for entity

### PR DESCRIPTION
**Pull request by BohdanHamurda**

### CODE REVIEW

**Review by TsisinskyjV — APPROVED** [review]

File: Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/Controllers/StudentController.cs [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+                }
+            }
+            catch (Exception)
+            {
+                return StatusCode(StatusCodes.Status500InternalServerError);
+            }
+        }
+
+
+        [HttpPut("{id}")]
```
Comment (bvzayats) [inline Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/Controllers/StudentController.cs]: I think you will have not to do in Adjust routing task. So I recommend you to left implementation and adjusting another endpoint from Get All for the next task.

Comment (bvzayats) [reply Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/Controllers/StudentController.cs]: Do it and it will be Ok

Comment (BohdanHamurda) [reply Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/Controllers/StudentController.cs]: Done

File: Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/Controllers/StudentController.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                        return Ok(db.Students.ToList());
+                    else
+                        return NoContent();
+                }
+            }
+            catch (Exception)
+            {
+                return StatusCode(StatusCodes.Status500InternalServerError);
+            }
+        }
```
Comment (bvzayats) [inline Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/Controllers/StudentController.cs:28]: Left other endpoints and should them return null.

Comment (BohdanHamurda) [reply Bohdan_Khamurda/WebAPISwagger/WebAPISwagger/Controllers/StudentController.cs:28]: Done

**Review by FoxLviv — APPROVED** [review]

**Review by bvzayats — APPROVED** [review]

**Pull request merged**